queens/project-nova/nuage-nova-compute-dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM registry.access.redhat.com/rhosp13/openstack-nova-compute:latest
+MAINTAINER Nuage OpenStack Engineering opensource@nuagenetworks.net
+
+LABEL name="nuagenetworks-rhosp13/openstack-nova-compute-5-4-1U1"
```

Comment:
1) Change container namespace for all
2) Change the nova-compute container name. (AVRS)

Reply:
made the 1st change, for the 2nd one, I just recollected that we agreed on the name in dockerfile

Reply:
Coming to container name, the format should be nuagenetworks/rhosp13-openstack-neutron-server-<nuage-version>.